lib/interact-for-login.ts

```diff
@@ -21,7 +21,7 @@ export const interactForLogin = async () => {
 
   config.set(`${getServer()}.pat`, pat)
 
-  console.log(`PAT saved! You may not begin using the CLI!`)
+  console.log(`PAT saved! You may begin using the CLI!`)
```

Comment:
I suppose this is the message which was meant here?

Reply:
yes haha thanks, how did this happen...